docs/why-qlik-core.md

```diff
@@ -0,0 +1,52 @@
+# An analytics development platform by Qlik
+
+Qlik Core is built around the Qlik Associative Engine and Qlik-authored open source libraries.
+Who’s Qlik you might ask? We’ve been around for 25 years and our specialty is analytics.
+Our platform has powerful features like responsive and intelligent visualizations,
+collaboration and storytelling functionality, robust data control and role-based data security.
+Underneath those features is our "secret sauce" that we call the Qlik Associative Engine.
+This in-memory inference and calculation engine is what’s responsible for the magic —
+dynamically indexing and associating data without needing to rely on query-based analyses
+which are restricted to linear exploration within a partial view of the data.
+
+With Qlik Core, you can take advantage of this powerful associative engine to build,
+extend and quickly deploy custom interactive data-driven solutions that are highly scalable
+and cloud-ready. Supported by a backbone of Linux and Docker, Qlik Core applications can
+easily be embedded into any stack, IoT device, application, web page, etc. And with the
+associative engine, all data points are associated within a virtual data layer over your
+front-end and back-end systems. These associations exist even across tables, allowing you
```

Comment:
"And with the
associative engine, all data points are associated within a virtual data layer over your
front-end and back-end systems"

What does this mean?

Reply:
I'm also clueless here. Maybe remove this?

Reply:
@TRB3269 thoughts?